config/bowler.php

```diff
@@ -2,6 +2,12 @@
 
 return [
 
+    'log' => [
```

Comment:
[minor] better put the least used configurations at the bottom of the file and have the connection config (most accessed) at the top

Reply:
Yeah, sure.